client/blocks/comments/post-comment-list.jsx

```diff
@@ -210,6 +231,28 @@ class PostCommentList extends React.Component {
 							} )
 						}</span> : null }
 				</div> }
+				<SegmentedControl>
+					<SegmentedControlItem
+						selected={ commentsFilter === 'all' }
+						onClick={ this.handleFilterClick( 'all' ) }>{ translate( 'All' ) }
+					</SegmentedControlItem>
+					<SegmentedControlItem
+						selected={ commentsFilter === 'approved' }
+						onClick={ this.handleFilterClick( 'approved' ) }>{ translate( 'Approved' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Approved', { context: 'comment status'} )` *ES Score: 20*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/filter-comments-on-posts)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).